propagator/xray/README.md

```diff
@@ -0,0 +1,43 @@
+# opentelemetry-propagator-xray
+
+The `opentelemetry-propagator-xray` gem contains injectors and extractors for the
+[AWS XRay propagation format][aws-xray].
+
+## What is OpenTelemetry?
+
+[OpenTelemetry][opentelemetry-home] is an open source observability framework, providing a general-purpose API, SDK, and related tools required for the instrumentation of cloud-native software, frameworks, and libraries.
+
+OpenTelemetry provides a single set of APIs, libraries, agents, and collector services to capture distributed traces and metrics from your application. You can analyze them using Prometheus, Jaeger, and other observability tools.
+
+## How does this gem fit in?
+
+This gem can be used with any OpenTelemetry SDK implementation. This can be the official `opentelemetry-sdk` gem or any other concrete implementation.
+
+## How do I get started?
+
+Install the gem using:
+
+```
+gem install opentelemetry-propagator-xray
+```
+
+Or, if you use [bundler][bundler-home], include `opentelemetry-propagator-xray` in your `Gemfile`.
```

Comment:
This should also describe how to configure the use of this propagator with the SDK. (I realize we're missing that in the README for the B3 propagator 😞.)

Reply:
Bump